app/scss/components/_header.scss

```diff
@@ -33,5 +33,17 @@ header {
       font-weight: bold;
       margin: 0;
     }
+
+    @media screen and (max-width: 960px) {
```

Comment:
Sorry I wasn't clear. At the top of _base.scss, I defined this mixin:
```
@mixin breakpoint($point) {
  @if $point == medium-screen {
    @media (max-width: 960px) { @content; }
  }
  @else if $point == small-screen {
    @media (max-width: 600px) { @content; }
  }
}
```
So whenever you need to add a breakpoint, use this 
```
@include breakpoint(medium-screen) {
    // new styles
  }
```

Instead of this:
```
 @media screen and (max-width: 960px) {
  // new styles
 }
```